docs/csharp/quick-starts/arrays-and-collections.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Quick Start - Arrays and Collections - C# Guide
+description: Learn C# in your browser
```

Comment:
This isn't an "in your browser" one right?

Reply:
fixed